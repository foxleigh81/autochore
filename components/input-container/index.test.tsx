/* global it expect */

import React from 'react';
import { render } from '@testing-library/react';

// Import component files
import InputContainer from './index';

// Tests
describe('InputContainer component', () => {
  it('renders to match snapshot', () => {
    const { baseElement } = render(
      <InputContainer inputName="Test Name" labelText="Test Input">
        <input type="text" />
      </InputContainer>
    );
    expect(baseElement).toMatchSnapshot();
  });
});
