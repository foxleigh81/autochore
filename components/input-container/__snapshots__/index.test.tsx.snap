// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputContainer component renders to match snapshot 1`] = `
<body>
  <div>
    <div
      class="input-container"
    >
      <label
        class="input-label"
        for="Test Name"
      >
        Test Input
      </label>
      <input
        type="text"
      />
    </div>
  </div>
</body>
`;
